editor/filesystem_dock.cpp

```diff
@@ -2039,14 +2039,16 @@ void FileSystemDock::_file_option(int p_option, const Vector<String> &p_selected
 
 				// Edit node in Tree.
 				tree->grab_focus();
-				tree->edit_selected(true);
+				if (to_rename.path != "res://") {
```

Comment:
An early return (or break in this case) would be better here.
```suggestion
				if (to_rename.path == "res://") {
					break;
				}
```